bom/pom.xml

```diff
@@ -500,6 +500,36 @@
                 <artifactId>helidon-common-tls</artifactId>
                 <version>${helidon.version}</version>
             </dependency>
+            <dependency>
+                <groupId>io.helidon.codegen</groupId>
```

Comment:
I do prefer the co-location by feature here (e.g. inject is a feature, and it has its own codegen). 
e.g. `helidon-inject-codegen` rather than `helidon-codegen-inject` (as I probably use the `inject` feature, and this is my `codegen` for that feature.
Regardless of the choice here, I expect us to have some kind of "bundle" of Helidon codegens that could be used by customers for their project, without the need to understand details (e.g. `helidon-bundle-apt`  or something like that). 

I do not think there is a single "right" answer for this choice.